pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -93,26 +94,60 @@ workflow RNAWithUMIsPipeline {
         r1_fastq = select_first([r1_fastq]),
         r2_fastq = select_first([r2_fastq]),
         bam_filename = output_basename,
-        library_name = select_first([library_name]),
-        platform = select_first([platform]),
-        platform_unit = select_first([platform_unit]),
-        read_group_name = select_first([read_group_name]),
-        sequencing_center = select_first([sequencing_center])
+        library_name = library_name,
+        platform = platform,
+        platform_unit = platform_unit,
+        read_group_name = read_group_name,
+        sequencing_center = sequencing_center
     }
   }
 
   File bam_to_use = select_first([bam, FastqToUbam.unmapped_bam])
```

Comment:
no longer needed, can just use `FastqToUbam.unmapped_bam`

Reply:
I don't think so - you can still provide either bam or fastqs as top level inputs. It's just that later in the pipeline he's making fastqs for the fastqc task and so needs the values like platform, etc to be defined.

Reply:
ohhhh okay that makes sense